Hi3Helper.Core/Classes/Data/Tools/GenshinDispatchHelper/GenshinDispatchHelper.cs

```diff
@@ -26,7 +26,7 @@ public class GenshinDispatchHelper : IDisposable
         public GenshinDispatchHelper(int RegionID, string DispatchKey, string DispatchURLPrefix, string VersionString = "2.6.0", CancellationToken cancelToken = new CancellationToken())
         {
             if (RegionID >= 4) ChannelName = "CNRELWin";
-            this._httpClient = new Http.Http(false, 1, 1);
+            this._httpClient = new Http.Legacy.Http(false, 1, 1);
```

Comment:
This is unimportant but if we dont really need this to be super fast or anything, I think we can move this to c#'s own httpclient, just less codes that needs to be monitored